src/pageObject.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Header {
+  constructor() {
+    this.logo = 'Conduit logo';
+  }
 
+  getLogo() {
+    return this.logo;
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.slug = slug;
+    this.url = url + slug;
+    this.commentButton = '[Publish comment] button';
```

Comment:
should contain getter property `commentButton`, so u should create getter method